processing/src/main/java/org/apache/druid/segment/column/LongsColumn.java

```diff
@@ -75,6 +77,13 @@ public long getLongSingleValueRow(int rowNum)
     return column.get(rowNum);
   }
 
+  @Override
+  @Nullable
+  public <T> T as(Class<? extends T> clazz)
+  {
+    return column.as(clazz);
```

Comment:
The class does not use SemanticUtils.makeAsMap so it would not be directly overridable. I think the decision of using overrides or not currently belongs to the class itself, in this way. The extension can still override it by adding an override for ColumnarLongs which this class is pointing to.